lib/kernel.coffee

```diff
@@ -15,7 +15,7 @@ class Kernel
         console.log 'Kernel spec:', @kernelSpec
         console.log 'Kernel configuration:', @config
         console.log 'Kernel configuration file path:', @configPath
-        @language = @kernelSpec.grammarLanguage
+        @language = KernelManager::getLanguageFor @grammar
```

Comment:
This isn't working.

I'm guessing this is caused by the dependency-cycle this statement creates between `KernelManager` and `Kernel`.

Since `Kernel::language` and `Kernel::grammar` aren't used anywhere in the code, I think it's best we remove them and update the code [here](https://github.com/lgeiger/hydrogen/blob/9a9726a677b8b91322457134717bd3b886dd8ee6/lib/kernel.coffee#L14), [here](https://github.com/lgeiger/hydrogen/blob/9a9726a677b8b91322457134717bd3b886dd8ee6/lib/kernel.coffee#L14), [here](https://github.com/nteract/hydrogen/blob/079608feca61a9c419cbcc4445cfc68252a271d7/lib/kernel-manager.coffee#L67) and [here](https://github.com/nteract/hydrogen/blob/079608feca61a9c419cbcc4445cfc68252a271d7/lib/kernel-manager.coffee#L76).